server/pulp/server/config.py

```diff
@@ -136,6 +136,10 @@ def _load_config(self):
         'download_interval': '30',
         'download_concurrency': '5'
     },
+    'profiling': {
+        'enabled': False,
```

Comment:
I think the tests are failing because this is `False` but it expects `'false'`. Crazy I know. Looking earlier in the file is the only reason I know. Please push a new version and I can kick off the tests again.